### PR TITLE
Asc alerts support

### PR DESCRIPTION
**Pull request by tamir-michaeli**
Added support for ASC alerts api.

### CODE REVIEW

**Comment by tamir-michaeli** [comment]
Fixed failed commit 6728e77 with commit c9f6df0.

File: README.md [hunk 1]
```diff
@@ -95,8 +104,11 @@ logLevel: INFO
...
-| targetApis.ADApis | **Default**: `300` <br>  List of AD apis to run. Required at least 1 api to run. Current supported apis: directoryAudits, signIns, riskySignIns. All apis are case sensitive and should be configured as mentioned here.|
+| targetApis | **Required**. <br> Specifies types of api lists to run, each API provider has its own list. Must contain at least 1 list with 1 api in the list. Current supported providers: ADApis, ASCApis. |
+| targetApis.ADApis | **Optional**. <br>  List of AD apis to run. Current supported apis: directoryAudits, signIns (this includes risky sign ins), riskySignIns. All apis are case sensitive and should be configured as mentioned here. |
+| targetApis.ASCApis | **Optional**. <br>  List of ASC apis to run. Current supported apis: alerts. All apis are case sensitive and should be configured as mentioned here. |
 | logLevel | **Default**: `INFO` <br> Log level for Logizo-MSGraph to omit. Can be one of: `OFF`, `ERROR`, `WARN`, `INFO`, `DEBUG`, `TRACE`, `ALL`. |
+| additionalFields | **Optional**. <br> List of additional fields to be added to the sent data. Pairs of key and value.
 
 #### <span id="if-fromdisk-true">If fromDisk=true</span>
```
Comment (Doron-Bargo) [inline README.md:114]: please add change log

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
All good - please ee my comment on readme and 1 question i had

File: src/main/java/main/MSClient.java [hunk 2]
```diff
@@ -105,6 +111,16 @@ private MSGraphConfiguration loadMSGraphConfig(String yamlFile) throws FileNotFo
         checkNotNull(config.getAzureADClient().getClientId(), "Parameter azureADClient.clientId is mandatory");
```
[the remark this answers is not in the document]
Comment (tamir-michaeli) [reply src/main/java/main/MSClient.java:111]: Yes. We use the same authentication process as the one for azure AD.

**Pull request merged**